GTBitsOfGood/national-npp
# 43 nonprofit projects frontend

## What is this?
Completed the nonprofit projects page and front-end is now fetching data from the back-end

## Steps to Test / Set Up
- If you are signed in, you'll need to add a nonprofit Id to your User document in the DB
- Play around in the DB with the statuses of the projects associated with each nonprofit to see how the updates are reflected on the UI 
- Play around with adding multiple projects with a nonprofit Id to see how multiple projects show up on the UI 

## Associated Issues
closes #43
